src/css/components/_layout.scss

```diff
@@ -1,7 +1,15 @@
 .video-js {
-  display: block;
+  /* inline-block is as close as we get to the video el's display:inline */
+  display: inline-block;
+  /* Make video.js videos align top when next to video elements */
+  vertical-align: top;
   box-sizing: border-box;
 
+  /* Default to the video element width/height. This will be overridden by
+   * the source width height unless changed elsewhere. */
+  width: 300px;
+  height: 150px;
```

Comment:
I've been shifting towards 16:9 as a default for awhile now. I think I'm convinced it's a better default setting at this point.